components/logins/android/src/main/java/mozilla/appservices/logins/LoginsStorageException.kt

```diff
@@ -56,3 +53,19 @@ class RequestFailedException(msg: String) : LoginsStorageException(msg)
  * This error is emitted if a sync or other operation is interrupted.
  */
 class InterruptedException(msg: String) : LoginsStorageException(msg)
+
+/**
+ * A reason a login may be invalid
+ */
+enum class InvalidLoginReason {
+    /** Hostnames may not be empty */
+    EMPTY_HOSTNAME,
```

Comment:
I really wish this was called `EMPTY_ORIGIN` to align with the upcoming naming so this isn't a breaking change soon.

Reply:
@mnoorenberghe I'll create another PR to correct this.

Reply:
Thank you! Unfortunately I wasn't subscribed on the PR so only saw it after merge.